less/styles.less

```diff
@@ -214,3 +214,10 @@ ul.tri-col {
   -moz-column-count: 3;
   column-count: 3;
 }
+
+.news-list {
```

Comment:
You will want to wrap this with another class since it's changing how we display news on the homepage:
https://deploy-preview-791--jakartaee.netlify.app/